monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -104,6 +104,26 @@ class ReportPageComponent extends AuthComponent {
     }
   }
 
+  addRansomwareReportTab() { // TODO: Fetch mode from API endpoint
+    let ransomwareTab = {key: 'ransomware', title: 'Ransomware report'};
+
+    // let mode = "";
+    // this.authFetch('/api/mode')
```

Comment:
```suggestion
    // this.authFetch('/api/island-mode')
```

Reply:
I think this should be changed once the API endpoint is added. It's just a comment right now (with a TODO so that we don't miss it).